kolibri/core/logger/serializers.py

```diff
@@ -27,6 +27,7 @@
 class ContentSessionLogSerializer(KolibriModelSerializer):
 
     extra_fields = serializers.JSONField(default="{}")
+    is_leaf = serializers.SerializerMethodField()
```

Comment:
I don't think `is_leaf` is required for this serializer.

Reply:
Yes, none of the changes in this file should be kept.

Reply:
Sorry, my bad. I must have left those while refactoring.